src/data/markdown/translated-guides/en/07 Misc/09 Glossary.md

```diff
@@ -1,189 +1,203 @@
+In the context of k6, virtual users can make concurrent requests as a test runs. In k6 cloud, you can also run multiple tests concurrently.</dd>
 
-### Endurance Testing
+<dt id="correlation">Correlation</dt>
+<dd>The process of taking [dynamic data](#dynamic-data) received from the system under test then reusing the data in a subsequent request. Testers commonly use correlation to retrieve and reuse session IDs or tokens for a [virtual user](#virtual-user)'s iteration lifetime.
+<p>Read more: [Correlation and dynamic data example](https://k6.io/docs/examples/correlation-and-dynamic-data/), [Correlation in testing APIs](https://k6.io/docs/testing-guides/api-load-testing/#correlation-and-data-parameterization)</p></dd>
 
-**Endurance testing** is a synonym for [soak testing](#soak-test).
+<dt id="checks">Checks</dt>
+<dd>Conditions that validate the correctness of a service.
+In k6 scripts, checks are a function that validates a response to a test request.
+<p>Read more: [Checks reference](/using-k6/checks)</p></dd>
 
-### Goja
+<dt id="duration">Duration</dt>
+<dd>The length of time that a test runs. When duration is set as an option, VU code runs for as many iterations as possible in the length of time specified (with possible finishing time given as a [graceful stop](#graceful-stop).
+<p>Read more: [Duration option reference](https://k6.io/docs/using-k6/k6-options/reference/#duration)</p></dd>
 
-**Goja** is a JavaScript runtime, purely written in go, that emphasizes standard compliance and performance. We use goja to allow for test scripting without having to compromise speed, efficiency or reliability, which would have been the case using NodeJS. For more details, see the [Goja repository on GitHub](https://github.com/dop251/goja).
+<dt id="dynamic-data">Dynamic data</dt>
+<dd>Data that might, or will, change between test runs. Common examples are order IDs, session tokens, or timestamps.
+<p>Read more: [Correlation and dynamic data example](https://k6.io/docs/examples/correlation-and-dynamic-data/)</p></dd>
 
-### Horizontal Scalability
+<dt id="endurance-testing">Endurance testing</dt>
+<dd>A synonym for [soak testing](#soak-test).</dd>
 
-**Horizontal Scalability** is a trait describing to what degree a system under test’s performance and capacity may be increased by adding more nodes (servers or computers for instance).
+<dt id="executor">Executor</dt>
+<dd>An property of a [scenario](#scenario) that configures VU behavior. You can use executors to configure whether to designate iterations as shared between VUs or to run per VU, or to configure or whether the VU concurrency is constant or changing</dd>
 
-### HTTP Archive
+<dt id="goja">Goja</dt>
+<dd>A JavaScript runtime, written in pure Go, that emphasizes standard compliance and performance. k6 uses Goja to enable test scripting without compromising speed, efficiency or reliability, compromises that would have been necessary with NodeJS
```

Comment:
A JavaScript runtime written in Go. k6 uses Goja to enable scripting in Javascript.